### PR TITLE
added max-change related stats in NaturalGradientAffineComponent

### PR DESCRIPTION
**Pull request by xiaohui-zhang**

### CODE REVIEW

**Comment by xiaohui-zhang** [comment]
ok fixed that

File: src/nnet3/nnet-simple-component.cc [hunk 1]
```diff
@@ -1308,12 +1343,34 @@ void NaturalGradientAffineComponent::Update(
   precon_ones.CopyColFromMat(in_value_temp, in_value_temp.NumCols() - 1);
 
   BaseFloat local_lrate = scale * minibatch_scale * learning_rate_;
+  update_count_ += 1;
```
Comment (danpovey) [inline src/nnet3/nnet-simple-component.cc]: make this say update_count_ += 1.0;
(and change any similar places)... always use floating-point constants in a floating-point context.
Also please rebase your changes so the fixes don't appear in the commit history.  rebase -i HEAD~3 and changing the last 2 lines from 'pick' to 'f' should do it (after you commit this fix).

**Comment by xiaohui-zhang** [comment]
fixed floating-point constants. Thanks

**Comment by danpovey** [comment]
cool.  I'll wait till checks complete and then merge.

**Pull request merged**